doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -1224,6 +1224,79 @@ Controlling the locality of commands
      occurs in a section.  The :cmd:`Set` and :cmd:`Unset` commands belong to this
      category.
 
+.. _controlling-typing-flags:
+
+Controlling Typing Flags
+----------------------------
+
+.. flag:: Guard Checking
+
+   This option can be used to enable/disable the guard checking of
+   fixpoints. Warning: this can break the consistency of the system, use at your
```

Comment:
If you wish, you can use `.. warning::` directives instead.

Reply:
Only if you add a non positive inductive ... This is debatable I admit.